releasenotes/sfdx/README.md

```diff
@@ -14,7 +14,20 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 Want to check out the new `sf` executable of Salesforce CLI? [Click here for the release notes.](../sf/README.md)
 
-## 7.144.2 (March 31, 2022) [stable-rc]
+## 7.145.0 (April 7, 2022) [stable-rc]
+
+These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
+
+* FIX: Setting your default Dev Hub while authorizing it now immediately propagates to your whole CLI environment. As a result, valid scratch org deletions no longer occasionally fail with error `Unable to associate this scratch org with a DevHub`. (GitHub issue [#1423](https://github.com/forcedotcom/cli/issues/1423))
+
+* FIX: The `force:source:*` commands now support these metadata types:
```

Comment:
Is this a FIX or a CHANGE ??

Reply:
Fix. In theory, all MD types should be supported from the get-go.  But as folks find types that aren't supported (bug) they add 'em to the registry.